homeassistant/components/meteo_france/const.py

```diff
@@ -77,7 +77,7 @@
         ENTITY_ICON: "mdi:weather-pouring",
         ENTITY_CLASS: "timestamp",
         ENTITY_ENABLE: True,
-        ENTITY_API_DATA_PATH: "current_forecast:T:value",
+        ENTITY_API_DATA_PATH: "NOT_USED",
```

Comment:
```suggestion
        ENTITY_API_DATA_PATH: None,
```

Forgot this one sorry

Reply:
OK